app/styles/custom/_utilities.scss

```diff
@@ -235,6 +235,10 @@
     font-weight: 500;
 }
 
+.text-italic{
```

Comment:
why is this needed? did we not have text italic before or inherited?

Reply:
This style is not inherited from bootstrap, thus the new addition.